src/BlockTemplatesController.php

```diff
@@ -84,15 +111,14 @@ public function get_block_templates() {
 			}
 
 			$new_template_item = array(
-				'title' => ucwords( str_replace( '-', ' ', $template_slug ) ),
+				'title' => BlockTemplateUtils::convert_slug_to_title( $template_slug ),
```

Comment:
Thank you, I appreciate you iterating on this! Definitely a nice win from a user perspective 🙂